### DISCUSSION TITLE
выдает ошибку и сразу же вылетает

### DISCUSSION THREAD

**Discussion by chelden23** [Flowseal/zapret-discord-youtube#1126] [Q&A]
![image](https://github.com/user-attachments/assets/aa158606-2817-492e-a365-3952629df576)
пишет что не может найти какието файлы
помогите плиз

**Comment by Flowseal** [comment]
Программа не может найти путь до файла "bin/winws.exe"
Либо его съел антивирус, либо вы неразархивировали всё
